src/examples/delete.yml

```diff
@@ -0,0 +1,21 @@
+description: |
+  Deploying kubernetes resources.
```

Comment:
Is this description accurate? This seems to delete a k8 resource.

Reply:
You are completely correct, I'll update that